spec/commands/check_spec.rb

```diff
@@ -15,10 +15,17 @@ def stub_json(uri, body)
       .to_return(headers: { 'Content-Type' => 'application/json' }, body: body.to_json)
   end
 
+  def stub_prs(uri, body)
+    stub_json(uri, body)
+    body.each do |pr|
+      stub_json(uri.sub('pulls', 'pulls/' + pr[:number].to_s).sub('&per_page=100', ''), pr)
```

Comment:
Hm, I've taken another look and I still think this is correct, but my Ruby's not very good - I'm trying to stub out all the PRs which are being passed in as they are requested one-by-one instead of stubbing out the list of PRs.  I have to remove the "per_page" portion of the URL because that's a parameter only passed to the list-of-PRs endpoint.  I could try doing it a little cleaner, like passing in a base url, and appending `pulls?direction-asc&per_page=100&sort=updated&state=open` to the first `stub_json` call and `'pulls/' + pr[:number].to_s` to the looped calls.  Is that better?